packages/mobile/src/import/ImportWallet.tsx

```diff
@@ -33,6 +33,11 @@ import UseBackToWelcomeScreen from 'src/onboarding/UseBackToWelcomeScreen'
 import { RootState } from 'src/redux/reducers'
 import { isAppConnected } from 'src/redux/selectors'
 
+const AVERAGE_WORD_WIDTH = 80
+const AVERAGE_SEED_WIDTH = AVERAGE_WORD_WIDTH * 24
+// Estimated number of lines needed to enter the account key
+const NUMBER_OF_LINES = Math.ceil(AVERAGE_SEED_WIDTH / Dimensions.get('window').width)
```

Comment:
We don't really use the whole width of the screen, so could this make the account key not fit entirely? I guess it's not a huge issue in any case, but just wondering :)

Reply:
True. This estimate is quite conservative though with 80 pt for each word.